transparency-certified/tro-utils
# Provide a set of default report templates

Provide a set of default templates as part of the package. They can be used by "name" instead of filepath.

### How to test?
1. Deploy this branch
2. Confirm that `tro-utils report --help` displays: 
    ```
    Options:
      -t, --template STRING_OR_PATH  Template file or one of the following:
                                     default  [required]
    ```
3. Using an example TRO, run report with `-t default -o new.html`
4. Download `curl -LO https://raw.githubusercontent.com/craig-willis/trace-report/main/templates/tro.md.jinja2`
5. Using the same example TRO, run report with `-t tro.md.jinja2 -o old.html`
6. Check that `new.html` is the same as `old.html`.
